tests/integration_tests.rs

```diff
@@ -364,3 +364,14 @@ fn memspike() {
 fn memory_stress_test() {
     run_test("memory_stress_test", 0, None, None, None);
 }
+
+#[test]
+#[serial]
+fn check_amd_sev_chain() {
+    if Path::new("/dev/sev").exists() {
+        assert!(
+            Path::new("/var/cache/amd-sev/chain").exists(),
+            "`/var/cache/amd-sev/chain` does not exist"
+        );
+    }
+}
```

Comment:
Use the [`cached_chain` module](https://github.com/enarx/sev/blob/master/src/util/cached_chain.rs#L64) to adhere to the Enarx search path for the cert.